install.md

```diff
@@ -182,7 +182,7 @@ Then to install Buildah on Ubuntu follow the steps in this example:
   export GOPATH=`pwd`
   git clone https://github.com/containers/buildah ./src/github.com/containers/buildah
   cd ./src/github.com/containers/buildah
-  PATH=/usr/lib/go-1.10/bin:$PATH make runc all TAGS="apparmor seccomp"
+  PATH=/usr/lib/go-1.8/bin:$PATH make runc all SECURITYTAGS="apparmor seccomp"
```

Comment:
should this still be 1.10 and not back to 1.8?

Reply:
Yes